pythonExecSource/src/main/python/pyExecConnector.py

```diff
@@ -659,6 +666,7 @@ class Connectors:
     def __init__(self):
         self.connector_set = VantiqConnectorSet()
         self.logger = logging.getLogger('Vantiq.PyExecConnector')
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
Is this just debugging or supposed to be committed?

Reply:
It's early enough in the process (and, if used at all, used by our folks) that I'd prefer to get more data now.  I'll mark it as a TODO & remove it after things get burned in a bit.